[CHERRY-PICK] MinPlatformPkg: Skip FSP libraries for non FSP build

PaddyDengAmi opened the pull request:
## Description

Some platforms consuming MinPlatformPkg does not use FSP architecture. Therefore does not have IntelFsp2WrapperPkg presents.

- [ ] Impacts functionality?
- [ ] Impacts security?
- [ ] Breaking change?
- [ ] Includes tests?
- [ ] Includes documentation?

## How This Was Tested

Build passed in project that does not have IntelFsp2WrapperPkg

## Integration Instructions

N/A

PaddyDengAmi committed:
[CHERRY-PICK] MinPlatformPkg: Skip FSP libraries for non FSP build

Some platforms consuming MinPlatformPkg does not use FSP architecture.
Therefore does not have IntelFsp2WrapperPkg presents.

Signed-off-by: PaddyDeng <paddydeng@ami.com>
(cherry picked from commit be53a3b9f1cc2ac92ee55d6e97ca911c00528b5f)

makubacki commented:
@PaddyDengAmi, I selected "Backport to release branch?" since I'd assume that you want this in a release branch. Please confirm if that is the case.

makubacki commented on MinPlatformPkg/Include/Dsc/CorePeiLib.dsc:
You also need to cherry-pick this commit https://github.com/tianocore/edk2-platforms/commit/489c97220c613b5cf45bc088a507a731d485cb0a.

PaddyDengAmi replied:
Updated. Thanks for the reminder.

PaddyDengAmi committed:
[CHERRY-PICK] MinPlatformPkg: Fix build failure of undefined PCD.

Some builds failed with below error:
PcdFspWrapperBootMode is not defined in DSC file.

Fixing this issue by defining this PCD in MinPlatform common include
DSC file.

Signed-off-by: Chasel Chiu <chasel.chiu@intel.com>
(cherry picked from commit 489c97220c613b5cf45bc088a507a731d485cb0a)
